account_invoice_view_payment/tests/test_account_invoice_view_payment.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountInvoiceViewPayment(TransactionCase):
+    """
+        Tests for Account Invoice View Payment.
+    """
+    def setUp(self):
+        super(TestAccountInvoiceViewPayment, self).setUp()
+        self.par_model = self.env['res.partner']
+        self.acc_model = self.env['account.account']
+        self.inv_model = self.env['account.invoice']
+        self.inv_line_model = self.env['account.invoice.line']
+        self.pay_model = self.env['account.payment']
+        self.reg_pay_model = self.env['account.register.payments']
+
+        self.cash = self.env['account.journal'].create(
+            {'name': 'Cash Test', 'type': 'cash'})
+        self.payment_method_manual_in = self.env.ref(
+            "account.account_payment_method_manual_in")
+
+        self.partner1 = self._create_partner()
+
+        self.invoice_account = self.acc_model.search(
+            [('user_type_id',
+              '=',
+              self.env.ref('account.data_account_type_revenue').id
```

Comment:
@hveficent `self.env.ref('account.data_account_type_revenue').id` --> `self.ref('account.data_account_type_revenue')`

Reply:
it's correct as it is now. `self.env.ref()` return records, not ids

Reply:
@mreficent It's not blocking anywhere but @hveficent needed ID so use `self.ref('account.data_account_type_revenue')` It's better.
otherwise LGTM :+1: